Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -191,6 +191,20 @@ def validate_enforcement_boundary_create_arguments(
         raise InvalidValueError("protocol", protocol, VALID_PROTOCOLS)
 
 
+def validate_ip_lists_get_arguments(max_results: Optional[int]) -> None:
+    """Validate arguments for ip-lists-get command.
+
+    Args:
+        max_results: Number of maximum results returned.
+    """
+    if isinstance(max_results, int) and (max_results < 1):  # type: ignore
```

Comment:
Should we validate the `ip_address` field here as well?

Reply:
Yes we can